leaf-ui/js/onFunctions.js

```diff
@@ -94,9 +94,9 @@ $('#btn-view-assignment').on('click', function() {
  */
 $('#btn-save-assignment').on('click', function() {
     saveAbsoluteTimePoints();
-    saveRelativeIntentionAssignments();
     saveAbsoluteIntentionAssignments();
     saveAbsoluteRelationshipAssignments();
+    saveRelativeIntentionAssignments();
```

Comment:
I had presumed that since there wasn't previously a note of order that it was okay to change the order to reflect the actual order of the elements in the UI for clarity. I just double checked my reasoning, and I believe that although saveAbsoluteIntentionAssignments() and saveRelativeIntentionAssignments() both deal with a model's constraint list, since they handle different types of constraints there should not be an order issue, although please do let me know if I am misunderstanding the different Intention types. Should I note this in a comment?

Reply:
If the order doesn't matter, then no comment is necessary.